docs/4.3/aws_oss_guide.md

```diff
@@ -386,12 +379,308 @@ To upgrade to a newer version of Teleport:
 - Copy  and its contents should also be backed up and copied over to the new instance.
 - Either restart the instance, or log in via SSH and run `sudo systemctl restart teleport.service`
 
-# Running Teleport Enterprise on AWS
-Most of this guide has been designed for OSS Teleport. Most of this guide also applies to Teleport Enterprise
-with a few extra notes around adding a license and getting the correct binary. If you would
-like help setting up Teleport Enterprise on AWS, please mail us at info@gravitational.com
+## Accessing EKS using Teleport
+
+This guide is based on an orignial post from AWS Open Source Blog - [Authenticating to EKS Using GitHub Credentials with Teleport](https://aws.amazon.com/blogs/opensource/authenticating-eks-github-credentials-teleport/) but will be updated with latest Teleport version & best practices.
+
+### Prerequisites
+You’ll need a functioning EKS cluster, we recommend version 1.16.  If you’re unfamiliar
+with creating an EKS cluster, see [eksctl.io](https://eksctl.io). Make sure you have eksctl version 0.22.0.
+
+- EKS Version: The below guide has been tested with [Kubernetes 1.16](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+- [jq](https://stedolan.github.io/jq/) installed on you  local machine.
+- An AWS Account with Root Access
+
+### Create cluster role and role binding
+The first step is to create a cluster role and role binding that will allow the Teleport
+EC2 instance to impersonate other users, groups, and service accounts.
```

Comment:
nit: I think we generally leak too much implementation detail in our k8s docs. The user most likely doesn't care how Teleport works under the hood.
How about: "... allow the Teleport EC2 instance to relay user requests to the Kubernetes API"

Reply:
Thanks for the fresh perspective. added.